modules/oe_paragraphs_banner/src/BannerParagraphUpdater.php

```diff
@@ -62,6 +62,10 @@ public function updateParagraph(ParagraphInterface $paragraph): void {
         continue;
       }
 
+      // Clone the revision onto original to make sure all the revisions get
+      // updated.
+      $revision->original = clone $revision;
```

Comment:
Is the comment wrong or the fix? In case it's the fix please try without cause this is supposed to migrate the the values for revisions that have the same values as the default one. It's one of the issues reported in the ticket.

Reply:
Updated test coverage for testing this case: creating a new revision for the same data to make sure it fixes the problem.